engine/evd.mli

```diff
@@ -509,6 +509,9 @@ val loc_of_conv_pb : evar_map -> evar_constraint -> Loc.t option
 val evars_of_term : evar_map -> econstr -> Evar.Set.t
   (** including evars in instances of evars *)
 
+val evars_of_term2 : evar_map -> econstr -> Evar.Set.t
```

Comment:
This should be of type `constr -> Evar.Set.t` (the evar map argument isn't used so don't have it)